### ISSUE TITLE
[SerializedField] private don't show as MinMaxSlider if inherence has abstruct class in middle

### ISSUE THREAD

**Issue by laurentopia** [TylerTemp/SaintsField#79]
![image](https://github.com/user-attachments/assets/6cd386ac-3e21-4a7f-8efc-00f1ae0b2dcd)
show up as vector2
![image](https://github.com/user-attachments/assets/6d61335e-66ae-4423-9b35-c582e98da0c6)

**Comment by TylerTemp** [comment]
Hi,

More information and context would be appreciated as I can not re-produce it with Unity 2022.2.0f1 under either IMGUI or UI Toolkit with script:

```csharp
public class Issue79 : MonoBehaviour
{
    [MinMaxSlider(0, 20f), SerializeField] Vector2 range;  // test field

    [Serializable]
    public struct MyStruct
    {
        [MinMaxSlider(0, 20f), SerializeField] Vector2 range;  // test under struct
    }

    [SaintsRow] public MyStruct myStruct;
}
```

![image](https://github.com/user-attachments/assets/3ca95ed8-1f47-44a2-8eca-4c453fca2dbd)

**Comment by laurentopia** [comment]
the structure is
![image](https://github.com/user-attachments/assets/cbfef51a-4cad-4a1f-9715-4cf5d42b653b)
![image](https://github.com/user-attachments/assets/39327665-6c45-485f-a5ec-a01f7d7c4f12)
![image](https://github.com/user-attachments/assets/94555e91-0d93-415b-ba14-b25b49dec838)
so it's just an inheritance of an abstract class that inherit from a mono

**Comment by TylerTemp** [comment]
Bug confirmed. Will be fixed in next release

**Comment by TylerTemp** [comment]
Hi,

This bug is fixed in dev branch.

Because of my health issue I need to slow down a bit on this project. If this issue bothers you much, please considering using `dev` branch to get the most recent stable features & bug fixing

**Comment by TylerTemp** [comment]
Hi, this bug has been fixed in [3.3.0](https://github.com/TylerTemp/SaintsField/releases/tag/3.3.0). As it's a breaking changes, please read the changelog before upgrading.

**Comment by laurentopia** [comment]
great and that fixed the [Required] bug I was getting.
